docs/content/dagster-cloud/developing-testing/branch-deployments.mdx

```diff
@@ -21,19 +22,23 @@ Branch Deployments automatically create staging environments of your Dagster cod
 
 Think of a branch deployment as a branch of your data platform, one where you can preview changes without impacting production or overwriting a testing environment.
 
-<center>
-  <Image
-    src="/images/dagster-cloud/developing-testing/branch-deployments/branch-deployments.png"
-    width={532}
-    height={573}
-  />
-</center>
+<Image
+src="/images/dagster-cloud/developing-testing/branch-deployments/branch-deployments.png"
+width={2032}
+height={2160}
+/>
```

Comment:
this image is a bit large, can we make it smaller? it currently takes up the entire screen while scrolling.

Reply:
Just pushed up a smaller version - lmk if that works